homeassistant/components/roomba/sensor.py

```diff
@@ -63,3 +81,194 @@ def icon(self):
     def native_value(self):
         """Return the state of the sensor."""
         return self._battery_level
+
+
+class CleaningTime(IRobotEntity, SensorEntity):
+    """Class to hold Roomba Sensor basic info."""
+
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
```

Comment:
Have you considered setting [`entity_registry_enabled_default`](https://developers.home-assistant.io/docs/core/entity/?_highlight=has_entit#advanced-properties) to `false` for these sensors so that these are not shown on the UI by default, and therefore do not increase the DB size of people who won't care about these diagnostic statistics?

Reply:
That's an good idea. I will do that.